playbooks/roles/shadowsocks/tasks/main.yml

```diff
@@ -17,27 +11,23 @@
   command: cat {{ shadowsocks_password_file }}
   register: shadowsocks_password
 
-- name: Install Shadowsocks
-  pip: name=shadowsocks
+- name: Add the Shadowsocks-libev APT key
+  apt_key: url=http://shadowsocks.org/debian/1D27208A.gpg state=present
```

Comment:
The convention in Streisand is to include a local copy of the key in the repository and to add it using the `data` argument of the `apt_key` module. [Here](https://github.com/jlund/streisand/blob/master/playbooks/roles/nginx/tasks/main.yml#L12) is an example of how that is being done in the Nginx role.

This is particularly important for security in a situation like this where the APT key is being served over plaintext HTTP and is therefore highly susceptible to a MITM attack. There are speed and reliability advantages as well.